Formula/blucat.rb

```diff
@@ -17,6 +17,13 @@ def install
   end
 
   test do
-    system "#{bin}/blucat", "doctor"
+    begin
+      io = IO.popen(bin/"blucat", "scan", "0")
```

Comment:
@ilovezfs `Utils.popen` is rarely used directly. Most often we use `Utils.popen_read` and that one accepts a variable number of arguments. `IO.popen` just like `Utils.popen` expects the command and all its arguments to be either passed as a string (executed in a shell) or as an array of strings (executed without a shell).